Student/student.html

```diff
@@ -0,0 +1,37 @@
+<html>
+
+<head>
+    <script type="text/javascript" src="../../build/server.bundle.js"></script>
+</head>
+
+<body>
+    <div>
+        <dialog id="addStuDia">
+            <label>Name</label>
+            <input id="stuName">
+            <label>Grade</label>
+            <input id="stuGrade">
+            <button id="add" onclick="add(event)">Add</button>
+        </dialog>
+        <button id="addStudent">ADD STUDENT</button>
+    </div>
+
+    <ul id="studentUlId">
+    </ul>
+    <dialog id="dialogId">
+        <h1>Are you sure to delete the student?</h1>
+        <button id="removeStd" onclick="removeStudent()">YES</button>
+        <button onclick="closeDialog()">NO</button>
+    </dialog>
+    <dialog id="errorDialog">
+        <h1>Request failed</h1>
+        <button onclick="closeErrorDialog()">Close</button>
+    </dialog>
+    <dialog id="displayStudentDetailsId">
```

Comment:
`id` can be something specific like `displayStudentDetailsDlg`

Reply:
changed

Reply:
because to separate <li> and <button>. It is required while showing student details. Otherwise if we keep everything in <li>, <button> also will be shown in the popup.